storybookjs/storybook
# CLI: Remove random commas in storybook upgrade logs

Closes #22298



## What I did


Log the `stdout` and `stderr` properties, instead of logging `output` property of result returned by `npm-check-updates@latest`. 

This provides the following benefits:
- Avoids commas that come from logging an array, which solves #22298.
- It is clearer in the code what is being logged, since `stdout` and `stderr` are equal to `output[1]` and `output[2]` respectively.
- `stderr` can be logged at a different level to `stdout` (not sure what the rule for using info or warn on logs)

It also:
- Does not log `output[0]` which in this case is null.
- Adds an extra newline between `stdout` and `stderr` logs

An example of what the logs look like before and after with some forced warnings (using reproduction code):

![image](https://user-images.githubusercontent.com/16689477/235499426-b7680f15-6b25-4ad7-9dea-f9e0a4078058.png)

## How to test

1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template react-vite/default-ts`
2. Run `npx storybook@latest upgrade`
3. Observe logs, specifically for `npm-check-updates`

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
